app/renderer/about/bookmarks/bookmarks.js

```diff
@@ -176,11 +183,20 @@ class Bookmarks extends React.Component {
             allBookmarkFolders={this.state.bookmarkFolders}
             sortable={false}
             draggable={!this.state.search}
-            selectedFolderId={this.state.selectedFolderId} />
+            selectedFolderId={this.state.selectedFolderId}
+          />
         </div>
       </div>
     </div>
   }
 }
 
+const styles = StyleSheet.create({
+  bookmarksManager__header: {
```

Comment:
It will be deprecated with a component like `<siteDetailsPageHeader_withActions>`.